### PR TITLE
Add LNR token metadata

### PR DESCRIPTION
**Pull request by linguists**
# Add LNR token metadata

$LNR is the governance token of LiNEAR Protocol: https://linearprotocol.org/

## Mainnet

ERC-20: https://etherscan.io/address/0x802d89B6E511b335F05024a65161bcE7efc3f311
NEP-141: https://nearblocks.io/address/802d89b6e511b335f05024a65161bce7efc3f311.factory.bridge.near

## Testnet

ERC-20: https://goerli.etherscan.io/address/0x802d89B6E511b335F05024a65161bcE7efc3f311
NEP-141: https://testnet.nearblocks.io/address/802d89b6e511b335f05024a65161bce7efc3f311.factory.goerli.testnet

### CODE REVIEW

**Comment by linguists** [comment]
Question: As Goerli will retire soon, when will Rainbow Bridge support Sepolia?

File: tokens/lnr_testnet.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
...SpaceOnUse'%3E%3Cstop stop-color='%232FE6FF'/%3E%3Cstop offset='0.111111' stop-color='%2300DDFF'/%3E%3Cstop offset='0.222222' stop-color='%2300D3FF'/%3E%3Cstop offset='0.333333' stop-color='%2344C7FF'/%3E%3Cstop offset='0.444444' stop-color='%2373B9FF'/%3E%3Cstop offset='0.555556' stop-color='%239BA9FF'/%3E%3Cstop offset='0.666667' stop-color='%23BF96FF'/%3E%3Cstop offset='0.777778' stop-color='%23DC82EB'/%3E%3Cstop offset='0.888889' stop-color='%23F26CCC'/%3E%3Cstop offset='1' stop-color='%23FF58A8'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint3_linear_20_54' x1='250.049' y1='278.792' x2='512.695' y2='618.677' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%232FE6FF'/%3E%3Cstop offset='0.111111' stop-color='%2300DDFF'/%3E%3Cstop offset='0.222222' stop-color='%2300D3FF'/%3E%3Cstop offset='0.333333' stop-color='%2344C7FF'/%3E%3Cstop offset='0.444444' stop-color='%2373B9FF'/%3E%3Cstop offset='0.555556' stop-color='%239BA9FF'/%3E%3Cstop offset='0.666667' stop-color='%23BF96FF'/%3E%3Cstop offset='0.777778' stop-color='%23DC82EB'/%3E%3Cstop offset='0.888889' stop-color='%23F26CCC'/%3E%3Cstop offset='1' stop-color='%23FF58A8'/%3E%3C/linearGradient%3E%3C/defs%3E%3C/svg%3E%0A",
```
Comment (karim-en) [inline tokens/lnr_testnet.json:8]:
```suggestion
    "icon": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTAwMCIgaGVpZ2h0PSIxMDAwIiB2aWV3Qm94PSIwIDAgMTAwMCAxMDAwIiBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgo8cGF0aCBkPSJNNjAwLjE2NSA1MjkuNDVMNzEzLjM3NyA1NTkuNzg1TDc3Ny40MTggMzIwLjc4MUw1MTMuMjU2IDI0OS45OTlMNDgyLjkyMSAzNjMuMjEyTDYzMy44NzEgNDAzLjY1OEw2MDAuMTY1IDUyOS40NVoiIGZpbGw9IndoaXRlIi8+CjxwYXRoIGQ9Ik02MDAuMTY1IDUyOS40NUw3MTMuMzc3IDU1OS43ODVMNzc3LjQxOCAzMjAuNzgxTDUxMy4yNTYgMjQ5Ljk5OUw0ODIuOTIxIDM2My4yMTJMNjMzLjg3MSA0MDMuNjU4TDYwMC4xNjUgNTI5LjQ1WiIgZmlsbD0idXJsKCNwYWludDBfbGluZWFyXzIwXzU0KSIvPgo8cGF0aCBkPSJNNjAwLjE2NSA1MjkuNDVMNzEzLjM3NyA1NTkuNzg1TDc3Ny40MTggMzIwLjc4MUw1MTMuMjU2IDI0OS45OTlMNDgyLjkyMSAzNjMuMjEyTDYzMy44NzEgNDAzLjY1OEw2MDAuMTY1IDUyOS40NVoiIGZpbGw9InVybCgjcGFpbnQxX2xpbmVhcl8yMF81NCkiLz4KPHBhdGggZD0iTTM5Ny4yNTMgNDcwLjU0N0wyODQuMDQxIDQ0MC4yMTJMMjIwIDY3OS4yMTZMNDg0LjE2MiA3NDkuOTk3TDUxNC40OTcgNjM2Ljc4NUwzNjMuNTQ3IDU5Ni4zMzhMMzk3LjI1MyA0NzAuNTQ3WiIgZmlsbD0id2hpdGUiLz4KPHBhdGggZD0iTTM5Ny4yNTMgNDcwLjU0N0wyODQuMDQxIDQ0MC4yMTJMMjIwIDY3OS4yMTZMNDg0LjE2MiA3NDkuOTk3TDUxNC40OTcgNjM2Ljc4NUwzNjMuNTQ3IDU5Ni4zMzhMMzk3LjI1MyA0NzAuNTQ3WiIgZmlsbD0idXJsKCNwYWludDJfbGluZWFyXzIwXzU0KSIvPgo8cGF0aCBkPSJNMzk3LjI1MyA0NzAuNTQ3TDI4NC4wNDEgNDQwLjIxMkwyMjAgNjc5LjIxNkw0ODQuMTYyIDc0OS45OTdMNTE0LjQ5NyA2MzYuNzg1TDM2My41NDcgNTk2LjMzOEwzOTcuMjUzIDQ3MC41NDdaIiBmaWxsPSJ1cmwoI3BhaW50M19saW5lYXJfMjBfNTQpIi8+CjxkZWZzPgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MF9saW5lYXJfMjBfNTQiIHgxPSI0MS41MTQ0IiB5MT0iNzkuOTk5MyIgeDI9IjQ1MC4yNjIiIHkyPSI3ODcuMzI1IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MV9saW5lYXJfMjBfNTQiIHgxPSIyNTAuMDQ5IiB5MT0iMjc4Ljc5MiIgeDI9IjUxMi42OTUiIHkyPSI2MTguNjc3IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50Ml9saW5lYXJfMjBfNTQiIHgxPSI0MS41MTQ0IiB5MT0iNzkuOTk5MyIgeDI9IjQ1MC4yNjIiIHkyPSI3ODcuMzI1IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50M19saW5lYXJfMjBfNTQiIHgxPSIyNTAuMDQ5IiB5MT0iMjc4Ljc5MiIgeDI9IjUxMi42OTUiIHkyPSI2MTguNjc3IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8L2RlZnM+Cjwvc3ZnPgo=",
```

**Review by karim-en — CHANGES_REQUESTED** [review]

File: tokens/lnr.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
...SpaceOnUse'%3E%3Cstop stop-color='%232FE6FF'/%3E%3Cstop offset='0.111111' stop-color='%2300DDFF'/%3E%3Cstop offset='0.222222' stop-color='%2300D3FF'/%3E%3Cstop offset='0.333333' stop-color='%2344C7FF'/%3E%3Cstop offset='0.444444' stop-color='%2373B9FF'/%3E%3Cstop offset='0.555556' stop-color='%239BA9FF'/%3E%3Cstop offset='0.666667' stop-color='%23BF96FF'/%3E%3Cstop offset='0.777778' stop-color='%23DC82EB'/%3E%3Cstop offset='0.888889' stop-color='%23F26CCC'/%3E%3Cstop offset='1' stop-color='%23FF58A8'/%3E%3C/linearGradient%3E%3ClinearGradient id='paint3_linear_20_54' x1='250.049' y1='278.792' x2='512.695' y2='618.677' gradientUnits='userSpaceOnUse'%3E%3Cstop stop-color='%232FE6FF'/%3E%3Cstop offset='0.111111' stop-color='%2300DDFF'/%3E%3Cstop offset='0.222222' stop-color='%2300D3FF'/%3E%3Cstop offset='0.333333' stop-color='%2344C7FF'/%3E%3Cstop offset='0.444444' stop-color='%2373B9FF'/%3E%3Cstop offset='0.555556' stop-color='%239BA9FF'/%3E%3Cstop offset='0.666667' stop-color='%23BF96FF'/%3E%3Cstop offset='0.777778' stop-color='%23DC82EB'/%3E%3Cstop offset='0.888889' stop-color='%23F26CCC'/%3E%3Cstop offset='1' stop-color='%23FF58A8'/%3E%3C/linearGradient%3E%3C/defs%3E%3C/svg%3E%0A",
```
Comment (karim-en) [inline tokens/lnr.json:8]:
```suggestion
    "icon": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTAwMCIgaGVpZ2h0PSIxMDAwIiB2aWV3Qm94PSIwIDAgMTAwMCAxMDAwIiBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgo8cGF0aCBkPSJNNjAwLjE2NSA1MjkuNDVMNzEzLjM3NyA1NTkuNzg1TDc3Ny40MTggMzIwLjc4MUw1MTMuMjU2IDI0OS45OTlMNDgyLjkyMSAzNjMuMjEyTDYzMy44NzEgNDAzLjY1OEw2MDAuMTY1IDUyOS40NVoiIGZpbGw9IndoaXRlIi8+CjxwYXRoIGQ9Ik02MDAuMTY1IDUyOS40NUw3MTMuMzc3IDU1OS43ODVMNzc3LjQxOCAzMjAuNzgxTDUxMy4yNTYgMjQ5Ljk5OUw0ODIuOTIxIDM2My4yMTJMNjMzLjg3MSA0MDMuNjU4TDYwMC4xNjUgNTI5LjQ1WiIgZmlsbD0idXJsKCNwYWludDBfbGluZWFyXzIwXzU0KSIvPgo8cGF0aCBkPSJNNjAwLjE2NSA1MjkuNDVMNzEzLjM3NyA1NTkuNzg1TDc3Ny40MTggMzIwLjc4MUw1MTMuMjU2IDI0OS45OTlMNDgyLjkyMSAzNjMuMjEyTDYzMy44NzEgNDAzLjY1OEw2MDAuMTY1IDUyOS40NVoiIGZpbGw9InVybCgjcGFpbnQxX2xpbmVhcl8yMF81NCkiLz4KPHBhdGggZD0iTTM5Ny4yNTMgNDcwLjU0N0wyODQuMDQxIDQ0MC4yMTJMMjIwIDY3OS4yMTZMNDg0LjE2MiA3NDkuOTk3TDUxNC40OTcgNjM2Ljc4NUwzNjMuNTQ3IDU5Ni4zMzhMMzk3LjI1MyA0NzAuNTQ3WiIgZmlsbD0id2hpdGUiLz4KPHBhdGggZD0iTTM5Ny4yNTMgNDcwLjU0N0wyODQuMDQxIDQ0MC4yMTJMMjIwIDY3OS4yMTZMNDg0LjE2MiA3NDkuOTk3TDUxNC40OTcgNjM2Ljc4NUwzNjMuNTQ3IDU5Ni4zMzhMMzk3LjI1MyA0NzAuNTQ3WiIgZmlsbD0idXJsKCNwYWludDJfbGluZWFyXzIwXzU0KSIvPgo8cGF0aCBkPSJNMzk3LjI1MyA0NzAuNTQ3TDI4NC4wNDEgNDQwLjIxMkwyMjAgNjc5LjIxNkw0ODQuMTYyIDc0OS45OTdMNTE0LjQ5NyA2MzYuNzg1TDM2My41NDcgNTk2LjMzOEwzOTcuMjUzIDQ3MC41NDdaIiBmaWxsPSJ1cmwoI3BhaW50M19saW5lYXJfMjBfNTQpIi8+CjxkZWZzPgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MF9saW5lYXJfMjBfNTQiIHgxPSI0MS41MTQ0IiB5MT0iNzkuOTk5MyIgeDI9IjQ1MC4yNjIiIHkyPSI3ODcuMzI1IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50MV9saW5lYXJfMjBfNTQiIHgxPSIyNTAuMDQ5IiB5MT0iMjc4Ljc5MiIgeDI9IjUxMi42OTUiIHkyPSI2MTguNjc3IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50Ml9saW5lYXJfMjBfNTQiIHgxPSI0MS41MTQ0IiB5MT0iNzkuOTk5MyIgeDI9IjQ1MC4yNjIiIHkyPSI3ODcuMzI1IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8bGluZWFyR3JhZGllbnQgaWQ9InBhaW50M19saW5lYXJfMjBfNTQiIHgxPSIyNTAuMDQ5IiB5MT0iMjc4Ljc5MiIgeDI9IjUxMi42OTUiIHkyPSI2MTguNjc3IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiMyRkU2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjExMTExMSIgc3RvcC1jb2xvcj0iIzAwRERGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuMjIyMjIyIiBzdG9wLWNvbG9yPSIjMDBEM0ZGIi8+CjxzdG9wIG9mZnNldD0iMC4zMzMzMzMiIHN0b3AtY29sb3I9IiM0NEM3RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjQ0NDQ0NCIgc3RvcC1jb2xvcj0iIzczQjlGRiIvPgo8c3RvcCBvZmZzZXQ9IjAuNTU1NTU2IiBzdG9wLWNvbG9yPSIjOUJBOUZGIi8+CjxzdG9wIG9mZnNldD0iMC42NjY2NjciIHN0b3AtY29sb3I9IiNCRjk2RkYiLz4KPHN0b3Agb2Zmc2V0PSIwLjc3Nzc3OCIgc3RvcC1jb2xvcj0iI0RDODJFQiIvPgo8c3RvcCBvZmZzZXQ9IjAuODg4ODg5IiBzdG9wLWNvbG9yPSIjRjI2Q0NDIi8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0iI0ZGNThBOCIvPgo8L2xpbmVhckdyYWRpZW50Pgo8L2RlZnM+Cjwvc3ZnPgo=",
```

**Review by karim-en — CHANGES_REQUESTED** [review]

**Comment by karim-en** [comment]
> Question: As Goerli will retire soon, when will Rainbow Bridge support Sepolia?

Approximately within one month

**Review by karim-en — APPROVED** [review]

**Comment by linguists** [comment]
thanks! @karim-en do we have an estimation when the token metadata will be updated?

I think it's not ready yet. 

```bash
View call: 802d89b6e511b335f05024a65161bce7efc3f311.factory.bridge.near.ft_metadata()
{
  spec: 'ft-1.0.0',
  name: '',
  symbol: '',
  icon: null,
  reference: '',
  reference_hash: '',
  decimals: 0
}
```

**Comment by karim-en** [comment]
> thanks! @karim-en do we have an estimation when the token metadata will be updated?
> 
> I think it's not ready yet.
> 
> ```shell
> View call: 802d89b6e511b335f05024a65161bce7efc3f311.factory.bridge.near.ft_metadata()
> {
>   spec: 'ft-1.0.0',
>   name: '',
>   symbol: '',
>   icon: null,
>   reference: '',
>   reference_hash: '',
>   decimals: 0
> }
> ```

@linguists it is ready now

**Pull request merged**